fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/StatementRenderer.java

```diff
@@ -36,10 +37,12 @@
      * @param havingClause
      * @param orderByClause
      * @param paginationClause
+     * @param unionAll
+     * @param union
      * @return
      */
     T select(boolean distinct, SelectList selectList, FromClause fromClause, WhereClause whereClause, GroupByClause groupByClause, HavingClause havingClause,
-        OrderByClause orderByClause, PaginationClause paginationClause);
+        OrderByClause orderByClause, PaginationClause paginationClause, boolean unionAll, Select union);
```

Comment:
would it make sense to overload this so that we have a simpler option to call?

Reply:
This is only called in one place, and if we simplified the call, we'd have to create getters for the attributes in the Select object, which don't currently exist. I think we can defer until we need a simpler option.

Reply:
This is the renderer interface, so I think it's OK, because it's really just a framework call, not something used by consumers of the framework.